resources/keb/templates/authorization-policy.yaml

```
@@ -60,7 +60,14 @@ spec:
     from:
       - source:
           requestPrincipals:
+          {{- $oidcs := .Values.oidcs | default list }}
+          {{- if gt (len $oidcs) 0 }}
+          {{- range $i, $p := $oidcs }}
+          - {{ $p.issuer }}/*
+          {{- end }}
```

I would call it as `.Values.oidc.issuers`, already having a default values of empty list in the values.yaml (which has boolean value of false):

```suggestion
          {{- if .Values.oidc.issuers }}
          {{- range $i, $p := .Values.oidc.issuers }}
          - {{ $p}}/*
          {{- end }}
```